order/src/main/java/com/caldevsupplychain/order/model/Order.java

```diff
@@ -0,0 +1,55 @@
+package com.caldevsupplychain.order.model;
+
+import com.caldevsupplychain.common.entity.BaseEntity;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+import javax.persistence.*;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+@Data
+@Entity
+@EqualsAndHashCode(callSuper = true)
+@Table(name = "orders")
+public class Order extends BaseEntity {
+
+	@Column(name = "uuid", nullable = false, unique = true, updatable = false)
+	private String uuid;
+
+	@Column(nullable = false, updatable = false)
+	private Long user_id;
+
+	@Column(nullable = false, updatable = false)
+	private Long agent_id;
+
+	@Column(name = "SKU", nullable = false)
+	private String SKU;
+
+	@OneToMany(fetch = FetchType.EAGER)
```

Comment:
Use lazy fetch. Eager means every time you fetch order, all associated items will be fetched. 
This is a potential performance issue, for example if you query 3000 orders, each order has 3 items. You are fetching 9000 items back at the same time.